scripts/v2-migration/actions/LegacyPooledStaking.blockV1.js

```diff
@@ -0,0 +1,22 @@
+const { runAction } = require('./utils');
+
+async function action({ legacyPooledStaking, signer, opts = {} }) {
+  const tx = await legacyPooledStaking.connect(signer).blockV1(opts);
```

Comment:
this to be removed?

based on discussion seems pointless if we simply delete the code.

Reply:
yep, nuke it. the removal of this function is already merged